include/envoy/router/router.h

```diff
@@ -674,6 +684,11 @@ class RouteEntry : public ResponseEntry {
    */
   virtual const envoy::api::v2::core::Metadata& metadata() const PURE;
 
+  /**
+   * @return TlsContextMatchCriteria* the tls context match criterion for this route.
```

Comment:
What does `nullptr` signify? Please add in comments.

Reply:
updated comment - let me know if it needs more refinement